gender_novels/corpus_gen.py

```diff
@@ -334,7 +376,14 @@ def get_publication_date_from_copyright(novel_text):
     """
     match = re.search(r"(COPYRIGHT\,*\s*) (\d{4})", novel_text, flags = re.IGNORECASE)
     if (match == None):
-        return None
+        match = re.search(r"\d{4}", novel_text[:3000])
+        if (match != None):
+            year = int(match.group(0))
+            if (year < 2000 and year > 1492):
```

Comment:
why 2000?  there may be newer copyright entries in Gutenberg, so 2020 might be valid in a few years when this code is still active.  Why not give this code at least a 20-year lifespan:

```
if 1492 < year < 2038:
     ...
```